### PR TITLE
SFS-4696: Add Heartbeat For Diagnostics Aggregation

### PR DESCRIPTION
**Pull request by imjaya**
Added Heartbeats for Autobin
cameras.

**Merge with:**
https://github.com/MisoRobotics/chippy/pull/1503
https://github.com/MisoRobotics/fiducials/pull/89
https://github.com/MisoRobotics/simple-chef/pull/152
https://github.com/MisoRobotics/grillbot/pull/2076
https://github.com/MisoRobotics/fanuc-drivers/pull/263
https://github.com/MisoRobotics/miso-robot-driver-stack/pull/131
https://github.com/MisoRobotics/miso_realsense/pull/79
https://github.com/MisoRobotics/miso_peripherals/pull/146
https://github.com/MisoRobotics/miso_core/pull/382

### CODE REVIEW

File: nodes/usb_cam_node.cpp [hunk 1]
```diff
@@ -356,6 +363,8 @@ int main(int argc, char **argv)
 {
   ros::init(argc, argv, "usb_cam");
   usb_cam::UsbCamNode a;
+  a.heartbeat_updater_.setHardwareID("none");
+  a.heartbeat_updater_.add(a.heart_beat_);
```
Comment (nardavin) [inline nodes/usb_cam_node.cpp:367]: Set this in the usbcamnode constructor

**Review by nardavin — CHANGES_REQUESTED** [review]

File: nodes/usb_cam_node.cpp (hunk 1 above)
Comment (jp7hong) [reply nodes/usb_cam_node.cpp]: Modified. Thanks.

**Review by zzv2 — APPROVED** [review]

**Review by jp7hong — APPROVED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ project(usb_cam)
 ## Find catkin macros and libraries
 ## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
 ## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS image_transport roscpp std_msgs std_srvs sensor_msgs camera_info_manager)
+find_package(catkin REQUIRED COMPONENTS image_transport roscpp std_msgs std_srvs sensor_msgs camera_info_manager misocpp)
```
Comment (zzv2) [inline CMakeLists.txt:7]: i think you need to add the new dep to the package.xml as well

Comment (imjaya) [reply CMakeLists.txt:7]: Done.

**Review by nardavin — APPROVED** [review]

**Pull request merged**